pages/add/form/validator.js

```diff
@@ -36,11 +36,16 @@ const Validator = {
   },
   imageTypeValidator() {
     return {
-      error: `Only JPEG, PNG, GIF, or SVG file is allowed.`,
+      error: `Only JPG, JPEG, PNG, GIF, or SVG file is allowed.`,
```

Comment:
"file is" -> "files are", and rather than "allowed" we probably want to say "accepted".